mobile/src/main/res/drawable/ic_nfc_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
```

Comment:
Why not make this 180dp (don't forget to rename the file to omit the size suffix ;-) ) and make the ImageView wrap_content? I think this would be better for Android < 5.0, which uses PNGs Android Studio generates from this image.

Reply:
I haven't thought about that, changed! :)